CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+### Added
+
+- Added a `--force` flag to the `satellite update` command, which forces a satellite to sleep before starting the update process.
```

Comment:
```suggestion
- Added a `--force` flag to the `satellite update` command, which forces a satellite to sleep before starting the update process. This may forcibly kill ongoing builds currently running on the satellite.
```